### ISSUE TITLE
Decrypt Failed

### ISSUE THREAD

**Issue by FishOnHead** [rmcrackan/Libation#791]
After successfully scanning my library, Libation cannot complete the decryption process for titles. Liberating the title will enter it into the queue and begin downloading, but will be interrupted; the log states that the file was only partially downloaded and the decrypt failed.

This happens repeatedly on the same titles, as well as various other titles. I have tested to see if certain types of titles have the effect, but thus far all titles (roughly 15 or so) have given this error.

On a single occasion, after failing to decrypt three times, one book was successfully liberated/converted.

Each time the decrypt fails, it produces the same generic message in the Log:

```
Download &amp; Decrypt Step, Completed: [Title]
Download &amp; Decrypt: Decrypt failed
ERROR. All books have not been processed. Most recent book: processing failed
```
Upon viewing the Yellow indicator it provides the message:
`Title has been at least partially downloaded Click to complete`
and clicking results in the same repeated error sequence

Windows 10 (64-bit), 11.1.0 Classic

[_Log202311.zip](https://github.com/rmcrackan/Libation/files/13284209/_Log202311.zip)
[_Log202311_001.zip](https://github.com/rmcrackan/Libation/files/13284207/_Log202311_001.zip)

**Comment by rmcrackan** [comment]
So sorry about this. Aside from some login errors, Most of what I'm seeing in the log looks like network issues. Can you try on another network and see if you get the same error? A friends wifi, coffee shop, etc?

**Comment by FishOnHead** [comment]
Makes sense that it's network issues. I'm not in a situation at the moment where I can test it, but I will be tomorrow.

I switched to Chardonnay to see if there was any potential change and realized the progress bar told me a lot of the story; each attempt downloads a little further before failing, and with enough tenacity it will eventually succeed. Provided it doesn't freeze and stop progressing before the error, which requires a reload of the program since I can't clear the queue at that point.

I'm not sure _why_ it's network issues, as I've downloaded far more substantial things on this network, but it does align with my experience so far. I'll test tomorrow properly and then close the ticket if I can make a better determination.
